File: Backend/Models/Inventario.cs
namespace InventarioMultiSucursal.Api.Models;

public class Inventario
{
    public int Id { get; set; }
    public int ProductoId { get; set; }
    public int SucursalId { get; set; }
    public decimal Cantidad { get; set; }
    public decimal StockMinimo { get; set; }
    public decimal CostoPromedio { get; set; }
    public DateTime UpdatedAt { get; set; }

    public Producto? Producto { get; set; }
    public Sucursal? Sucursal { get; set; }
}

File: Frontend/src/types/inventario.ts
export interface InventarioItem {
  id: number
  productoId: number
  productoNombre: string
  productoSku: string
  unidadMedidaAbreviatura: string
  sucursalId: number
  cantidad: number
  stockMinimo: number
  costoPromedio: number
  updatedAt: string
}
